### ISSUE TITLE
UI Changes

### ISSUE THREAD

**Issue by srilaasya** [abhinavsri360/GDSC-Lead-Map#78]
Standardize the social icons and change the style of the button
Add footer

**Comment by abhinavsri360** [comment]
Made UI changes via 79d65b1
Footer yet to be implemented.

**Comment by aravindvnair99** [comment]
In dark mode the contrast is bad. Cards remain white while the background turns dark. Maybe we can take inputs from https://material.io/design/color/dark-theme.html?

**Comment by abhinavsri360** [comment]
I too feel that the contrast is bad, moreover, I added a package through which the mode change is happening, will figure out some ways sometime soon.

**Comment by aravindvnair99** [comment]
@abhinavsri360 That package doesn't seem to be changing anything in the cards at present. Maybe if it goes to a lighter shade of grey, it should look fine

**Comment by abhinavsri360** [comment]
@aravindvnair99 Yeah, even I am planning to not use the package and code it right away.

**Comment by abhinavsri360** [comment]
I think the issue is resolved now via ed3c5c65e4ef731001b50e0e879eab163751908d 
wdyt? @srilaasya @aravindvnair99

**Comment by aravindvnair99** [comment]
@abhinavsri360 Looks better now. It doesn't automatically pick though based on OS setting. And doesn't persist through reload as well. Previously it did.

**Comment by abhinavsri360** [comment]
Cool, I'll set state using local storage for this.
